src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsFandomUap.java

```diff
@@ -18,10 +19,9 @@
   public void adsFandomArticleUapDesktop(String pageType, String pageName, long atfId, long btfId) {
     AdsFandomObject fandomPage = loadPage(pageName, pageType);
     fandomPage.triggerOnScrollSlots();
-    verifyUapAtf(atfId, AdsFandomContent.TOP_BOXAD, fandomPage);
+    verifyUapAtf(atfId, AdSlot.TOP_BOXAD, fandomPage);
     verifyUapBtf(btfId,
-                 AdsFandomContent.INCONTENT_BOXAD,
-                 AdsFandomContent.BOTTOM_LEADERBOARD,
+                 AdSlot.INCONTENT_BOXAD,
```

Comment:
Below the fold part of UAP should have both: incontent box ad and bottom leaderboard.

Reply:
On some pages we've got `INCONTENT_BOXAD` but on another it's `FEED_BOXAD`. To have the tests consistent we need to make UAP slots names on news&stories consistent :/